enum/Analytics.ts

```diff
@@ -77,4 +80,28 @@ export const Events: Record<string, IAnalyticsEvent> = {
 			close_method: 'timeout',
 		},
 	},
+	[EventName.LIVEAGENT_SESSION_EXPIRED]: {
+		category: EventCatagory.CHAT_SESSION,
+		eventType: EventType.SESSION,
+		action: 'closed',
+		properties: {
+			close_method: 'expired_session',
+		},
+	},
+	[EventName.UNKNOWN_ERROR_IN_CHECKING_AGENT_RESPONSE]: {
+		category: EventCatagory.CHAT_SESSION,
+		eventType: EventType.SESSION,
+		action: 'failed',
```

Comment:
the `action: 'failed',` should be `closed` here, right? Or there any change for this particular event?